acceptance-tests/templates/prometheus-pm.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.podMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  labels:
+    {{- include "acceptance-tests.labels" . | nindent 4 }}
+  name: {{ include "acceptance-tests.fullname" . }}-pm
+spec:
+  podMetricsEndpoints:
+  - interval: 1d
```

Comment:
don't you think we would want to have some flexibility on the interval?

Reply:
I was wondering, but since the job is running once per day ... if we want it configureable, then we should find a way to have both variables configured in a coherent manner at the same time

Reply:
added an extra value to make the scrape interval configureable